### PR TITLE
Pagination update

### LINKED ISSUE #411: update the pagination component
Some changes have been made to the pagination component as seen in Figma: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=6530%3A2477

Should be pretty self-explanatory

### PR DESCRIPTION
**Pull request by drillprop**
Fix #411 
Figma: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=6530%3A2477

### CODE REVIEW

File: src/shared/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -27,33 +26,39 @@ const Pagination: React.FC<PaginationProps> = ({
...
+      onChangePage(idx === 1 ? internalPage - 2 : internalPage)
+    }
+  }
+
   return (
     <PaginationWrapper>
-      <PaginationButton
-        isHidden={internalPage <= 1}
-        isChevron={true}
+      <ChevronButton
```
Comment (mikkio-j) [inline src/shared/components/Pagination/Pagination.tsx:37]: in designs icon has `gray[50]` color when `:active`. Or maybe it has that styles only on the right side at all times. Not sure about that.

Comment (drillprop) [reply src/shared/components/Pagination/Pagination.tsx:37]: set default colors of the `<Button />`. Should be better now

**Review by mikkio-j — APPROVED** [review]

File: src/shared/components/Pagination/Pagination.style.tsx [hunk 2]
```diff
@@ -1,50 +1,67 @@
...
 `
 
 const buttonActiveState = css`
   color: ${colors.white};
   background-color: ${colors.gray[800]};
 `
 
+export const ChevronButton = styled(Button)`
+  width: ${sizes(12)};
+  height: ${sizes(12)};
```
Comment (kdembler) [inline src/shared/components/Pagination/Pagination.style.tsx]: I believe those should already be specified by the button component?

File: src/shared/components/Pagination/Pagination.style.tsx [hunk 3]
```diff
@@ -1,50 +1,67 @@
...
+    &:first-of-type {
+      margin-right: ${sizes(8)};
+    }
+    &:last-of-type {
+      margin-left: ${sizes(8)};
+    }
+  }
+`
+
 export const PaginationButton = styled.button<PaginationButtonProps>`
```
Comment (kdembler) [inline src/shared/components/Pagination/Pagination.style.tsx:51]: The background-color we have don't seem to be matching what's in Figma. We should use those: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=6530%3A2651

First for inactive item, second for hover, third for active state (so both current page and pressed button)

Comment (kdembler) [inline src/shared/components/Pagination/Pagination.style.tsx:51]: The text color seems to be white when item is active, should be `gray[50]`

Screenshot 2021-04-08 at 11 05 03

**Review by kdembler — CHANGES_REQUESTED** [review]
After testing it out, I don't think the dots should be clickable. If I'm on page 1 and I click dots, I wouldn't expect it to have the same function as "next" button. Let's make them just visual and non-interactive

File: src/shared/components/Pagination/Pagination.style.tsx (hunk 3 above)
Comment (drillprop) [reply src/shared/components/Pagination/Pagination.style.tsx:51]: Everything looks fine on Figma where we have a grayish background, but if we set these transparent colors on the black background - active buttons will be barely visible at least for my taste. I'll update this, but want to just raise the issue.
image

Comment (kdembler) [reply src/shared/components/Pagination/Pagination.style.tsx:51]: Good point, I'll raise this with the designers. For now, let's use what's in the design

**Review by kdembler — APPROVED** [review]

**Pull request merged**